### PR TITLE
Keras models support

### PR DESCRIPTION
**Pull request by typhoonzero**
Also removed python3.8 test because of this error: https://travis-ci.com/sql-machine-learning/models/jobs/202072631

Related work: https://github.com/sql-machine-learning/sqlflow/pull/424

### CODE REVIEW

File: tests/base.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class BaseTestCases:
+    class BaseTest(object):
+        def setUp(self):
+            self.model, self.features, self.label = None, {}, None
+
+        def test_train_and_predict(self):
+            self.setUp()
+            train_input_fn(self.features, self.label)
+
+            print('Calling BaseTest:testCommon')
```
Comment (tonyyang-svail) [inline tests/base.py]: Should we also test `self.model.fit` and `self.model.predict` here?

File: sqlflow_models/dnnclassifier.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        """Default optimizer name. Used in model.compile."""
+        return tf.keras.optimizers.Adagrad(lr=0.1)
+
+    def default_loss(self):
+        """Default loss function. Used in model.compile."""
+        return 'sparse_categorical_crossentropy'
+
+    def default_training_epochs(self):
+        """Default training epochs. Used in model.fit."""
+        return 2
```
Comment (tonyyang-svail) [inline sqlflow_models/dnnclassifier.py:38]: ~~`2` might be too small. Change to `50`?~~

**Review by tonyyang-svail — CHANGES_REQUESTED** [review]

File: sqlflow_models/dnnclassifier.py (hunk 2 above)
Comment (typhoonzero) [reply sqlflow_models/dnnclassifier.py:38]: Tested 2 is enough for iris demo case. Change to 50 maybe take too long to train

**Review by wangkuiyi — DISMISSED** [review]

File: tests/base.py (hunk 1 above)
Comment (typhoonzero) [reply tests/base.py]: Done.

**Review by tonyyang-svail — APPROVED** [review]
LGTM!

**Pull request merged**